doc/.custom_wordlist.txt

```diff
@@ -1,45 +0,0 @@
-ACL
```

Comment:
We tend to keep the history clean as there is no squashing performed to the commits when we merge the PR.

Can you please remove this file from the commit entirely as it's anyway added back in the commit after?

Reply:
Done :)